src/components/SignUp/SignUp.jsx

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable */
```

Comment:
Can we revert the changes on this file. I don't think these changes are required.